site/src/components/select-anatomy.jsx

```diff
@@ -6,13 +6,8 @@ const SelectAnatomy = () => {
     <AnatomyWrapper>
       <Host name="select">
         <Slot name="button">
-          <Part name="button">
-            <Slot name="selected-value">
-              <Part name="selected-value">Currently selected value</Part>
-            </Slot>
-            <Slot name="marker">
-              <Part name="marker">Dropdown indicator (e.g. icon)</Part>
-            </Slot>
+          <Part name="button type=selectlist">
+            <Part name="selectedvalue">Currently selected value</Part>
```

Comment:
Should the part name here be selectedvalue or just value?

Reason I ask is virtually all inputs will have some form of "value" part. But `selectedvalue` only really makes sense for elements with multiple choices? Saying that most elements can be linked up to a datalist so maybe `selectedvalue` is fine for all of them?

Reply:
The new `<selectedvalue>` element is only intended to work for `<selectlist>`, at least for now, and it is an element with multiple choices.

Do you think the element's tag name should be `<value>` instead?

Reply:
If it were to be value then it likely could be reused for other inputs in future?

Reply:
Good point. I opened an issue to discuss this further: https://github.com/openui/open-ui/issues/808